commands/acl-setuser.md

```diff
@@ -17,6 +17,18 @@ set command (+set rule). Then another SETUSER call can modify the user rules:
 
 The above rule will not apply the new rule to the user virginia, so other than `SET`, the user virginia will now be able to also use the `GET` command.
 
+Starting from Redis 7.0, ACL rules can also be grouped into multiple distinct sets of rules, called selectors.
+Selectors are added by wrapping the rules in parentheses and providing them just like any other rule.
+In order to execute a command, either the root permissions (rules defined outside of parenthesis) or any of the selectors (rules defined inside parenthesis) must match the given command.
+For example:
+
+    ACL SETUSER virginia on +GET allkeys (+SET ~app1*)
```

Comment:
Is "on" a pure token, then perhaps spell it in uppercase?

Reply:
No, it's one of the ACL rules. Which, in this file at least are never uppercased.